ccsdspy/packet_types.py

```diff
@@ -153,6 +153,10 @@ def load(self, file, include_primary_header=False):
            fields are: `CCSDS_VERSION_NUMBER`, `CCSDS_PACKET_TYPE`,
            `CCSDS_SECONDARY_FLAG`, `CCSDS_SEQUENCE_FLAG`, `CCSDS_APID`,
            `CCSDS_SEQUENCE_COUNT`, and `CCSDS_PACKET_LENGTH`
+        reset_file_obj : bool
+           If True, leave the file object, when it is file buffer, where it was before load is called.
+           Otherwise, (default), leave the file stream pos after the read packets.
+           Does not apply when file is a file location.
```

Comment:
Minor wording adjustment, I'd just say "Does not apply when file is a **string**" I had to double take for a second to figure out what file location exactly meant